src/requirements.yml

```
@@ -20,16 +18,16 @@ roles:
     src: https://github.com/cisagov/ansible-role-crowdstrike
   - name: freeipa_client
     src: https://github.com/cisagov/ansible-role-freeipa-client
-  - name: harden
-    src: https://github.com/cisagov/ansible-role-hardening-2
+  - name: konstruktoid.hardening
+    version: v2.0.4
+    src: https://github.com/konstruktoid/ansible-role-hardening.git
+    scm: git
```

Alphabetize and simplify to remove unnecessary `scm` attribute:
```suggestion
  - name: konstruktoid.hardening
    src: https://github.com/konstruktoid/ansible-role-hardening
    version: v2.0.4
```